src/libluau.zig

```diff
@@ -11,13 +11,19 @@ const c = @cImport({
 
 const config = @import("config");
 pub const lang = config.lang;
+/// True if Luau built for 4-vectors
+pub const luau_vector_size = config.luau_vector_size;
```

Comment:
I think the doc comment here needs updating because `luau_vector_size` is a usize, not a bool

Reply:
Fixed